ResearchKit/Charts/ORKGraphChartView.h

```diff
@@ -106,20 +109,6 @@ ORK_AVAILABLE_DECL
 - (NSInteger)graphChartView:(ORKGraphChartView *)graphChartView numberOfPointsForPlotIndex:(NSInteger)plotIndex;
```

Comment:
Ok, I'll go for `numberOfDataPointsForPlotIndex:`.

As for the types, they're currently named `ORKValueRange` and `ORKValueStack`. Perhaps we don't want to add the `DataPoint` suffix, as these types are internally chart-agnostic (they could be potentially reused outside charts). `DataPoint` refers more to its behavior inside a chart.

What I could do is rename the base `ORKGraphChartType` protocol to `ORKValueCollectionType`.

What do you reckon?